js/okex3.js

```diff
@@ -2168,28 +2168,28 @@ module.exports = class okex3 extends Exchange {
                 symbol = market['symbol'];
             }
         }
-        let amount = this.safeNumber (order, 'size');
-        const filled = this.safeNumber2 (order, 'filled_size', 'filled_qty');
+        let amount = this.safeString (order, 'size');
+        const filled = this.safeString2 (order, 'filled_size', 'filled_qty');
         let remaining = undefined;
         if (amount !== undefined) {
             if (filled !== undefined) {
-                amount = Math.max (amount, filled);
-                remaining = Math.max (0, amount - filled);
+                amount = Precise.stringMax (amount, filled);
+                remaining = Precise.stringMax (0, Precise.stringSub (amount, filled));
             }
         }
         if (type === 'market') {
-            remaining = 0;
+            remaining = '0';
         }
-        let cost = this.safeNumber2 (order, 'filled_notional', 'funds');
-        const price = this.safeNumber (order, 'price');
-        let average = this.safeNumber (order, 'price_avg');
+        let cost = this.safeString2 (order, 'filled_notional', 'funds');
+        const price = this.safeString (order, 'price');
+        let average = this.safeString (order, 'price_avg');
         if (cost === undefined) {
             if (filled !== undefined && average !== undefined) {
-                cost = average * filled;
+                cost = Precise.stringMul (average, filled);
             }
         } else {
             if ((average === undefined) && (filled !== undefined) && (filled > 0)) {
-                average = cost / filled;
+                average = Precise.stringDiv (cost, filled);
```

Comment:
Missed the `filled > 0` comparison on line 2191. We have to use `Precise.stringGt (filled, '0')` there.